src/mame/drivers/goldstar.cpp

```diff
@@ -17671,6 +17712,38 @@ void wingco_state::init_luckylad()
 	}
 }
 
+void cmaster_state::init_chthree()
+{
+	// Address swapping in 0x10 bytes blocks.
+	// 01234567-89abcdef <> c9ba8dfe - 41320576
+ 
+	uint8_t *ROM = memregion("maincpu")->base();
+	uint8_t row; 
+	uint16_t addr;
```

Comment:
Please scope these variables appropriately, and use `const` where it makes sense.